api/src/services/feature-flag.service.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  Injectable,
+  BadRequestException,
+  NotFoundException,
+} from '@nestjs/common';
+import { JurisdictionService } from './jurisdiction.service';
+import { PrismaService } from './prisma.service';
+import { FeatureFlag } from '../dtos/feature-flags/feature-flag.dto';
+import { FeatureFlagAssociate } from '../dtos/feature-flags/feature-flag-associate.dto';
+import { FeatureFlagCreate } from '../dtos/feature-flags/feature-flag-create.dto';
+import { FeatureFlagUpdate } from '../dtos/feature-flags/feature-flag-update.dto';
+import { SuccessDTO } from '../dtos/shared/success.dto';
+import { mapTo } from '../utilities/mapTo';
+
+/**
+      this is the service for feature flags
+      it handles all the backend's business logic for reading/writing/deleting feature flag data
+    */
+@Injectable()
+export class FeatureFlagService {
+  constructor(
+    private prisma: PrismaService,
+    private jurisdictionService: JurisdictionService,
+  ) {}
+
+  /**
+        this will get a set of feature flags
+      */
+  async list(): Promise<FeatureFlag[]> {
```

Comment:
should this be jurisdiction specific? right now it would return all flags across all jurisdictions instead of all flags on a jurisdiction

although maybe I'm wrong on how this will be used

Reply:
This does not need to be jurisdiction specific. Use case for this is we want to associate a jurisdiction with flags, we call all flags to see which we want to associate with